src/Microsoft.ML.Transforms/doc.xml

```diff
@@ -7,8 +7,7 @@
         Encodes the categorical variable with hash-based encoding. 
       </summary>
       <remarks>
-        CategoricalHashOneHotVectorizer converts a categorical value into an indicator array by hashing the
-        value and using the hash as an index in the bag.
+        CategoricalHashOneHotVectorizer converts a categorical value into an indicator array by hashing the value and using the hash as an index in the bag.
```

Comment:
>the value [](start = 104, length = 9)

why removing the newline here, but not, say, in line 33?

Reply:
I am not sure why the text is broken in newlines identically how my text is structured for the page corresponding to the CategoricalHashOneHotVectorizer, and it is flowing differently for the CategoricalOneHotVectorizer, disregarding line indentations in the XML. 

Could be because the text in the other one is inside the <para> tags; and we should always use the para tags to keep the convenience formatting in the xml. 

links if curious:
[https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.categoricalonehotvectorizer?view=ml-dotnet&branch=smoke-test-preview](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.categoricalonehotvectorizer?view=ml-dotnet&branch=smoke-test-preview)

[https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.categoricalhashonehotvectorizer?view=ml-dotnet&branch=smoke-test-preview](https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.ml.transforms.categoricalhashonehotvectorizer?view=ml-dotnet&branch=smoke-test-preview)






---
In reply to: [205926140](https://github.com/dotnet/machinelearning/pull/592#discussion_r205926140) [](ancestors = 205926140)